language/stdlib/modules/libra_block.move

```diff
@@ -48,14 +49,20 @@ module LibraBlock {
         previous_block_votes: vector<address>,
         proposer: address
     ) acquires BlockMetadata {
-      // Can only be invoked by LibraVM privilege.
-      Transaction::assert(Transaction::sender() == 0x0, 33);
+        // Can only be invoked by LibraVM privilege.
+        Transaction::assert(Transaction::sender() == 0x0, 33);
 
-      process_block_prologue(round, timestamp, previous_block_votes, proposer);
+        process_block_prologue(round, timestamp, previous_block_votes, proposer);
 
-      // Currently distribute once per-block.
-      // TODO: Once we have a better on-chain representation of epochs we will make this per-epoch.
-      TransactionFee::distribute_transaction_fees<LBR::T>();
+        // Currently distribute once per-block.
+        // TODO: Once we have a better on-chain representation of epochs we will make this per-epoch.
+        TransactionFee::distribute_transaction_fees<LBR::T>();
+
+        // Once every X blocks update all validators configs
+        // TODO(valerini): figure out what condition to put here
+        if (get_current_block_height() % 10000 == 0) {
+            LibraSystem2::update_all_validator_info();
+        }
```

Comment:
do we want to add this, in this PR?

Reply:
Agree, let's make this change later... we'll have to agree on the time-frame for regular reconfigurations. I will leave a TODO here.